docs/architecture/containerized-lifecycle/design-develop-containerized-apps/build-aspnet-core-applications-linux-containers-aks-kubernetes.md

```diff
@@ -44,7 +44,7 @@ Verify that you've selected ASP.NET Core 5.0 as the framework. .NET 5.0 is inclu
 
 Notice Docker support is not enabled now, just to show it can be done after project creation.
 
-If you have any previous version of .NET Core, you can download and install the 3.1 version from <https://dotnet.microsoft.com/download>.
+If you have any previous version of .NET Core, you can download and install the specific version from <https://dotnet.microsoft.com/download>.
```

Comment:
This statement does not make any sense here. Please remove this line.

Reply:
Sure.